sdk/eventhub/event-hubs/test/sender.spec.ts

```diff
@@ -12,17 +12,13 @@ import dotenv from "dotenv";
 dotenv.config();
 
 describe("EventHub Sender", function(): void {
-  const service = { connectionString: process.env.EVENTHUB_CONNECTION_STRING, path: process.env.EVENTHUB_NAME };
-  const client: EventHubClient = EventHubClient.createFromConnectionString(service.connectionString!, service.path);
```

Comment:
Since we now support the static helper as well, lets revert the changes to this file and the sendEvents.ts sample file